docs/conversion.md

```diff
@@ -0,0 +1,64 @@
+This document outlines all the conversion details regarding `docker-compose.yaml` values to Kubernetes / OpenShift artifacts.
+
+| Value             | Version   | Kompose     | K8s / OpenShift                                                                                                                                                                                                                                                          | Notes                                                                                                                       |
+|-------------------|-----------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+|                   |           |             |                                                                                                                                                                                                                                                                          |                                                                                                                             |
+| Service           |           |             |                                                                                                                                                                                                                                                                          |                                                                                                                             |
+| build             |           | YES         | [BuildConfig](https://docs.openshift.com/enterprise/3.1/dev_guide/builds.html#defining-a-buildconfig)                                                                                                                                                                    | Converts, but local builds are not yet supported. See issue [97](https://github.com/kubernetes-incubator/kompose/issues/97) |
```

Comment:
Maybe mention that build is supported only for OpenShift.
For Kubernetes `build` in unsupported